console: Document oidc issuer url and tls config (#771)
redpanda-data/documentation: modules/upgrade/pages/deprecated/index.adoc

====

// This is not supported in cloud
ifndef::env-cloud[]
== Kubernetes

[.sortable]
|===
| Deprecated in  | Feature | Details

| Operator v2.2.3-24.2.X
| Migration path from the deprecated Cluster and Console custom resources.

> |The deprecated Cluster and Console custom resources will no longer be migrated to the new Flux-based version.

Reviewer: ```suggestion
|The deprecated Cluster and Console custom resources are no longer migrated to the new Flux-based version.
```

> | As part of our ongoing enhancements, the deprecated Cluster and Console custom resources will not be migrated to the new Flux-based version. In the future, these custom resources will leverage the same reconciler that is currently used by Redpanda, ensuring a consistent and unified experience across all resources.

Reviewer 2: I don't think we should document the intent to unify the Cluster resource. That initiative is aimed at helping cloud and we don't want to encourage self hosted users to start using the Cluster CRD. 

...We don't document anything about the Cluster or Console CRD for end users aside from this migration, right?

Reviewer 3: Thanks, Chris. Updated.

Reviewer: Thanks for this update. Docs does not ever announce future plans/updates, so good catch and glad to see it was removed.
